.github/workflows/lint.yml

```diff
@@ -8,22 +8,21 @@ on:
 
 jobs:
   lint:
-
     runs-on: ubuntu-22.04
 
     steps:
-    - uses: actions/checkout@v3
-    - name: Set up Python 3.x
-      uses: actions/setup-python@v4
-      with:
-        python-version: '3.x'
-    - name: Update pip
-      run: pip install --upgrade pip
-    - name: Install black and pylint
-      run: pip install black~=22.3 pylint~=2.13,!=2.13.6
-    - name: Check files are formatted with black
-      run: |
-        black --check .
-    - name: Run pylint
-      run: |        
-        pylint */
+      - uses: actions/checkout@v3
```

Comment:
Is this and the line below changed on purpose?

Reply:
This was done by the `prettier` git hook that I added, which standardises the format of, among other things, yaml files. Is that ok?

Reply:
yes